addons/binding/org.openhab.binding.icloud/README.md

```diff
@@ -0,0 +1,55 @@
+# iCloud Binding
+
+The iCloud Binding helps to integrate apple device information into openHAB.
+
+## Supported Things
+
+The following devices are known to work with this binding:
+* iPhone 6s
+* iPhone 5c
+* iPhone 7
+* iPad Air 2/2017/Pro
```

Comment:
The original set of rules was able to retrieve data from my brothers iWatch 2. I didn't test with the binding yet but it might be safe to add?

Reply:
Let´s wait until we know that it works with the binding; as data de-serialization is quite different there. Please let me know in case you test it & it works. 

thanks for your feedbacks!

Reply:
See my screenshot from before. The Watch Series 2 works as expected. All channels/items are filled with correct data.

Only limitation: For some reason the Watch 2 identifies itself as "Apple Watch", without any Version 2 information. The only way I found to identify the device is via the `devicemodel` field in the json response. Compare identifier "Watch2,3" here: https://www.theiphonewiki.com/wiki/Models